docs/topics/gradle.md

```diff
@@ -258,33 +258,27 @@ kotlin.stdlib.default.dependency=false
 
 The [`kotlin.test` API](https://kotlinlang.org/api/latest/kotlin.test/) is available for testing different Kotlin projects. 
 
-Add the corresponding dependencies on test libraries:
-
-* For `commonTest`, add the `kotlin-test-common` and `kotlin-test-annotations-common` dependencies.
-* For JVM targets, use `kotlin-test-junit` or `kotlin-test-testng` for the corresponding asserter implementation and annotations mapping.
-* For Kotlin/JS targets, add `kotlin-test-js` as a test dependency. 
+Add the dependency `kotlin-test` to the `commonTest` source set, and the Gradle plugin will infer the corresponding 
+test dependencies for each source set:
+* `kotlin-test-common` and `kotlin-test-annotations-common` for common source sets
+* `kotlin-test-junit`/`kotlin-test-junit-5`/`kotlin-test-testng` for JVM source sets
+* `kotlin-test-js` for Kotlin/JS source sets
```

Comment:
I don't think that details of dependency names for each platform are useful to readers given that now all should work with the single dependency.

Reply:
But may it worth to be mentioned for the common understanding? Let's ask @p7nov 's opinion

Reply:
This may be useful for readers who add these dependencies manually now. They'll learn what exactly is replaced with `kotlin("test")`. But I agree that it's not necessary.

Reply:
Not necessary but may be useful – then I'll leave it here if nobody is against.

Reply:
ok